<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Settings">
   <item>
    <widget class="QScrollArea" name="scrollArea_Settings">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent_Settings">
      <layout class="QVBoxLayout" name="verticalLayout_SettContent">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_SettHeader">
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="text">
            <string>&lt; 返回</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SettTitle">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_SettHeader">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl_FontTitleConfig">
         <property name="text">
          <string>题目字体大小 (预览):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderFontTitle">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_PreviewTitle">
         <property name="text">
          <string>题目字体预览 Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_FontBaseConfig">
         <property name="text">
          <string>选项/按钮字体大小 (预览):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderFontBase">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_PreviewBase">
         <property name="text">
          <string>选项预览 Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_Sett1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAutoSubmit">
         <property name="text">
          <string>单选/判断题点击选项直接判分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkConfirmExit">
         <property name="text">
          <string>返回主页时显示确认提示</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAutoNext">
         <property name="text">
          <string>答对后自动跳转下一题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkDarkMode">
         <property name="text">
          <string>夜间模式 (Dark Mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_FileListStats">
         <item>
          <widget class="QLabel" name="lbl_FileListLimit">
           <property name="text">
            <string>题目列表显示数量:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFileListLimit">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_Settings">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grpPaths">
         <property name="title">
          <string>Path Settings (路径设置)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_Paths">
          <item row="0" column="0">
           <widget class="QLabel" name="lblRepoPath">
            <property name="text">
             <string>题库(Repo):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editRepoPath">
            <property name="placeholderText">
             <string>Default (空则使用默认)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="btnBrowseRepo">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblDataPath">
            <property name="text">
             <string>数据(Data):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editDataPath">
            <property name="placeholderText">
             <string>Default (空则使用默认)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btnBrowseData">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>保存设置并返回主页</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_Settings">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
